packages/app-project/src/components/AuthModal/components/RegisterForm/RegisterFormContainer.js

```diff
@@ -0,0 +1,57 @@
+import { inject, observer } from 'mobx-react'
+import auth from 'panoptes-client/lib/auth'
+import { func, shape } from 'prop-types'
+import React, { Component } from 'react'
+
+import RegisterForm from './RegisterForm'
+
+class RegisterFormContainer extends Component {
+  constructor() {
+    super()
+    this.onSubmit = this.onSubmit.bind(this)
+  }
+
+  onSubmit(values, { setFieldError, setSubmitting }) {
+    const { authClient, store } = this.props
+    authClient.register(values)
+      .then(userResource => {
+        setSubmitting(false)
+        store.user.set(userResource)
+        this.props.closeModal()
+      })
+      .catch(error => {
+        setFieldError('password', error.message)
```

Comment:
this was set to the password field for the sign-in box as the nearest handy field help. we may need to do some parsing of the error message to cover valid email, taken username etc

Reply:
I copied and pasted the LoginForm code as a starter :-D